waspc/cli/src/Wasp/Cli/Interactive.hs

```diff
@@ -83,18 +98,30 @@ askToChoose question options = do
     showIndexedOptions = intercalate "\n" $ showIndexedOption <$> zip [1 ..] (NE.toList options)
       where
         showIndexedOption (idx, option) =
-          Term.applyStyles [Term.Yellow] indexPrefix
-            <> Term.applyStyles [Term.Bold] (showOption option)
-            <> (if isDefaultOption option then " (default)" else "")
-            <> showDescription option (length indexPrefix)
+          concat
+            [ indexPrefix,
+              optionName,
+              tags,
+              optionDescription
+            ]
           where
-            indexPrefix = showIndex idx <> " "
+            indexPrefix =
+              Term.applyStyles
+                ([Term.Yellow] <> whenDefault [Term.Bold, Term.Underline])
+                (showIndex idx)
+                <> " "
+            optionName = Term.applyStyles [Term.Bold] (showOption option)
+            tags = whenDefault (Term.applyStyles [Term.Yellow] " (default)")
+            optionDescription = showDescription option
+            whenDefault xs = if isDefaultOption option then xs else mempty
 
         showIndex i = "[" ++ show (i :: Int) ++ "]"
 
-        showDescription option indentLength = case showOptionDescription option of
+        showDescription option = case showOptionDescription option of
           Just description -> "\n" <> replicate indentLength ' ' <> description
           Nothing -> ""
+          where
+            indentLength = 6
```

Comment:
I preferred the look of the description being aligned below the title 🤷 

Now it's a bit to the right 
![Screenshot 2023-12-19 at 12 40 28](https://github.com/wasp-lang/wasp/assets/2223680/6e333fd8-f916-4054-b4f7-3701c6254571)

Reply:
Okay, it's obviously subjective, so I want insist on this 👍

Reply:
I'd maybe look at other CLI tools how they indent their titles and descriptions. 

I found this in Astro's CLI 
![Screenshot 2023-12-21 at 15 18 14](https://github.com/wasp-lang/wasp/assets/2223680/bf754eea-8d9f-4bc3-a1b9-00ad93c77cb5)

Reply:
Just a reminder, this is what it looks like with old indent:
![image](https://github.com/wasp-lang/wasp/assets/1536647/746f6609-e572-4aa0-bded-16291171de8e)

This does look harder to read to me, because all text is so crammed up. In the example above from Astro, they have newlines, so that solves the issue, detaches title from the text. We don't have that here, nor would it be a good fit.

That said, since it is not obvious that change I made is better (both to you and @sodic ), then I will revert it to previous state.

Reply:
Done!